scripts/release_notes.rb

```diff
@@ -66,12 +66,12 @@ def gather_issues(offset = 0)
 
 puts "### Release notes for #{@current_release_name}"
 
-grouped_issues = gather_issues.group_by { |issue| issue['category']['name'] rescue issue['project']['name'] }
+grouped_issues = gather_issues.group_by { |issue| "#{issue['project']['name']}#{ ' - ' + issue['category']['name'] if issue.key?('category') }" }
```

Comment:
Can we transform groups here instead of below?
```ruby
def header_from_issue(issue)
  project = issue['project']['name']
  return project unless issue.key?('category')

  category = issue['category']['name']
  case issue['project']
  when 'Smart Proxy'
    return project if category == 'Core'
  case 'SELinux'
    return project if category == 'General Foreman'
  end

  "#{project} - #{category}"
end
```